ArduCopter/AP_Arming.cpp

```diff
@@ -929,8 +938,24 @@ bool AP_Arming_Copter::disarm(const AP_Arming::Method method, bool do_disarm_che
     // disable gps velocity based centrefugal force compensation
     ahrs.set_correct_centrifugal(false);
     hal.util->set_soft_armed(false);
-
+    
```

Comment:
Please turn on trailing white-space detection on your editor and remove this

Reply:
Thanks for this @rishabsingh3003 !